Src/Presentation/OpenChat.API/appsettings.Development.json

```diff
@@ -1,9 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "Default": "Debug",
+      "Microsoft": "Debug",
+      "Microsoft.Hosting.Lifetime": "Debug"
```

Comment:
We should decide on proper Debug levels. (P.S. test comment)

Reply:
revert to default?
we can change them live depending on needs (not sure if there's one size fits all here)

Reply:
We can revert them to the default values. We should have enough information to pinpoint the next action to take.